### ISSUE TITLE
Is it possible to run static linked elf executeable in the current process?

### ISSUE THREAD

**Issue by ghost** [Cloaked9000/ElfLoader#1]

**Comment by Cloaked9000** [comment]
As in, overwriting the current address space as opposed to forking?

**Comment by ghost** [comment]
Yes

**Comment by Cloaked9000** [comment]
No, I'm afraid not sorry. It could probably be done by moving more of the loading logic into the loader (such as the writing of the binary to the process), but it's not currently possible.

I'm currently in bed but I might have a look at adding it over the weekend :)
